packages/core/lib/model/map-context.ts

```diff
@@ -39,7 +40,16 @@ export interface MapContextLayerWmts extends MapContextBaseLayer {
 export interface MapContextLayerWfs extends MapContextBaseLayer {
   type: "wfs";
   url: string;
-  featureType: string;
+  name: string;
```

Comment:
We should keep `featureType` for WFS, it's the official denomination

Reply:
This is still changed to `name`, is that intentional? :slightly_smiling_face:

Reply:
The changes were made, but they were not added to the commit. I have now included them, and everything should be updated.